c++/src/capnp/compat/json-test.c++

```diff
@@ -22,7 +22,7 @@
 #include "json.h"
 #include <capnp/test-util.h>
 #include <capnp/compat/json.capnp.h>
-#include <capnp/compat/json-test.capnp.h>
+#include <capnp/json-test.capnp.h>
```

Comment:
This change presumably breaks the builds -- I guess your meson build is putting this file in the wrong place?